src/Parser/MarksmanshipHunter/Modules/Features/AimedInVulnerableTracker.js

```diff
@@ -32,14 +32,14 @@ class AimedInVulnerableTracker extends Module {
     this.totalAimed += 1;
   }
   suggestions(when) {
-    const percentAimedOutsideVulnerable = 100/this.totalAimed * this.outsideVulnerabilityAimed;
-    when(percentAimedOutsideVulnerable).isGreaterThan(2)
+    const percentAimedOutsideVulnerable =(100/this.totalAimed * this.outsideVulnerabilityAimed)/100;
```

Comment:
Sorry to be nitpicky but the 100's cancel out so we're left with `this.outsideVulnerabilityAimed / this.totalAimed` anyway, please update it for readability's sake and I think I can merge that :)